imagemounter/volume_system.py

```diff
@@ -73,6 +73,21 @@ def _make_subvolume(self, **args):
         self.volumes.append(v)
         return v
 
+    def _make_exfat_superfloppy(self):
```

Comment:
Agreed, it is not necessary.  I do have an example of an exFAT superfloppy that has issues, but this was not the way to handle it.  It has been removed.  See the comment re: _detect_exfat_superfloppy() for further details.